### PR TITLE
Update conftest for recent Grand Challenge development changes

### PR DESCRIPTION
**Pull request by jmsmkn**
`docker compose` doesn't work on GitHub actions right now, see comments.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -45,15 +45,28 @@ def local_grand_challenge() -> Generator[str, None, None]:
                 "scripts/development_fixtures.py",
                 "scripts/algorithm_evaluation_fixtures.py",
                 "scripts/image10x10x10.mha",
+                "scripts/minio.py",
```
Comment (chrisvanrun) [inline tests/conftest.py:48]: I assume an unrelated fix for the problem at hand?

File: tests/conftest.py [hunk 2]
```diff
@@ -122,4 +138,22 @@ def rewrite_docker_compose(content: bytes) -> bytes:
...
     return yaml.safe_dump(spec).encode("utf-8")
+
+
+def rewrite_makefile(content: bytes) -> bytes:
+    # Using `docker compose` with version 2.4.1+azure-1 does not seem to work
+    # It works locally with version `2.5.1`, so for now go back to docker-compose
+    # If this is fixed docker-compose-wait can be removed and the `--wait`
+    # option added to the "up" action above
+    makefile = content.decode("utf-8")
+    makefile = makefile.replace("docker compose", "docker-compose")
```
Comment (chrisvanrun) [inline tests/conftest.py:158]: This will work for now, but care should be taken for in some configurations/OS/frameworks `docker-compose` is just an alias for `docker compose`.

**Review by chrisvanrun — APPROVED** [review]
LGTM. Small note of care on aliasing.

File: tests/conftest.py (hunk 1 above)
Comment (jmsmkn) [reply tests/conftest.py:48]: This is now required for `make development_fixtures`, was changed a couple of weeks ago.

File: tests/conftest.py (hunk 2 above)
Comment (jmsmkn) [reply tests/conftest.py:158]: Yep, we should hear about it if it breaks.
